src/WebJobs.Script/Binding/HttpBinding.cs

```diff
@@ -77,7 +77,12 @@ public override Collection<CustomAttributeBuilder> GetCustomAttributes(Type para
                         if (jo.TryGetValue("status", StringComparison.OrdinalIgnoreCase, out value) && value is JValue)
                         {
                             statusCode = (HttpStatusCode)(int)value;
-                        } 
+                        }
+
+                        if (jo.TryGetValue("statusCode", StringComparison.OrdinalIgnoreCase, out value) && value is JValue)
```

Comment:
needed to add `statusCode` so that the response object could use the express `status(...)` function